homeassistant/components/buienradar/sensor.py

```diff
@@ -774,6 +774,16 @@ def _load_data(self, data):  # noqa: C901
         self._measured = data.get(MEASURED)
         sensor_type = self.entity_description.key
 
+        extra_state_attributes = {
+            ATTR_ATTRIBUTION: data.get(ATTRIBUTION),
```

Comment:
You can also just set `_attr_attribution = ".."` on the base entity

Reply:
setting self._attr_attribution now